src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -632,6 +642,22 @@ private string GetLanguagesToken(string mediaInfoLanguages)
             return string.Join("+", tokens.Distinct());
         }
 
+        private void UpdateMediaInfoIfNeeded(string pattern, Dictionary<string, int> minimumMediaInfoSchemaRevisions, EpisodeFile episodeFile, Series series)
+        {
+            var matches = TitleRegex.Matches(pattern);
+
+            var shouldUpdateMediaInfo = matches.Cast<Match>()
+                .Select(m => minimumMediaInfoSchemaRevisions.GetValueOrDefault(m.Value, -1))
+                .Where(r => r > -1)
```

Comment:
Don't really need the Where, do we?
Could even combine it into a single .Any, but main point should be readability.

Reply:
no we don't 👍